homeassistant/components/ruckus_unleashed/config_flow.py

```diff
@@ -65,18 +70,32 @@ async def async_step_user(self, user_input=None):
                 errors["base"] = "cannot_connect"
             except InvalidAuth:
                 errors["base"] = "invalid_auth"
-            except Exception:  # pylint: disable=broad-except
-                _LOGGER.exception("Unexpected exception")
```

Comment:
We normally have a catch of broad exception in config flows

Reply:
Done in https://github.com/home-assistant/core/pull/99411